partner-manager/modules/ROOT/pages/partner-manager-tutorial-td.adoc

```diff
@@ -170,22 +166,22 @@ include::_partials/tutorial/buyer/create-buyer-partner.adoc[]
 
 === Add Buyer Identifiers
 
-NOTE: Make sure you are configuring *AcmeHomeSupply*, not *AAZFasteners*.
+NOTE: Make sure you are configuring *AcmeHomeSupply*, not *GearCo*.
 
 ==== Add Buyer X12-ISA Identifier
 
 To add this identifier, perform steps in <<Add Supplier X12-ISA Identifier>>,
-substituting *Acme* for *AAZ*.
+substituting *Acme* for *GearCo*.
 
 ==== Add Buyer AS2 Identifier
 
 To add this identifier, perform steps in <<Add Supplier AS2 Identifier>>,
-substituting *Acme* for *AAZ*.
+substituting *Acme* for *GearCo*.
 
 === Upload the Buyer Certificate
 
 To add the certificate, perform steps in <<Upload the Supplier Certificate>>,
-substituting *partner.cer* for *HomeOrg.p12*.
+substituting *partner.pem* for *HomeOrg.p12*.
```

Comment:
Is the file extension really .pem? Just checking - I haven't heard of that one.

Reply:
Yes, the new certificates that I have got from Devang, it has .pem extension.